cmd/calico-node/main.go

```diff
@@ -82,9 +85,15 @@ func main() {
 		}
 	}
 
+	// Check for liveness
+	if *felixLive {
+		health.Run(*birdReady, *bird6Ready, *felixReady, *felixLive, *thresholdTime)
+		os.Exit(0)
+	}
+
 	// If any of the readienss options are provided, check readiness.
 	if *birdReady || *bird6Ready || *felixReady {
-		readiness.Run(*birdReady, *bird6Ready, *felixReady, *thresholdTime)
+		health.Run(*birdReady, *bird6Ready, *felixReady, false, *thresholdTime)
```

Comment:
Seems a bit odd to have two blocks here when we already have the relevant bools in hand.  Maybe just do:
```
if *felixLive || *birdReady || *bird6Ready || *felixReady { 
    health.Run(*birdReady, *bird6Ready, *felixReady, *felixLive, *thresholdTime)
    os.Exit(0)
}
```

Reply:
Fixed.